crates/node/src/cli/sync.rs

```diff
@@ -62,6 +62,13 @@ pub struct SyncParams {
     #[clap(env = "MADARA_N_BLOCKS_TO_SYNC", long, value_name = "NUMBER OF BLOCKS")]
     pub n_blocks_to_sync: Option<u64>,
 
+    /// Gracefully shutdown Madara once it has finished to synchronize all
+    /// blocks. This can either be once the node has caught up with the head of
+    /// the chain or when it has synced as many blocks as specified by
+    /// `n-blocks-to-sync`.
+    #[clap(env = "MADARA_STOP_ON_SYNC", long, value_name = "STOP ON SYNC", default_value_t = false)]
```

Comment:
The value_name isn't used for bool flags - you can see that in the `--help` output.